pkg/provider/aws/resources/eks.go

```diff
@@ -387,6 +386,25 @@ func (nodeGroup *EksNodeGroup) Configure(params EksNodeGroupConfigureParams) err
 	return nil
 }
 
+func (cluster *EksCluster) SetUpDefaultNodeGroup(dag *core.ResourceGraph, appName string) error {
+	_, err := core.CreateResource[*EksNodeGroup](dag, EksNodeGroupCreateParams{
+		InstanceType: "t3.medium",
+		NetworkType:  PrivateSubnet,
+		Refs:         cluster.ConstructsRef,
+		AppName:      appName,
+		ClusterName:  strings.TrimLeft(cluster.Name, fmt.Sprintf("%s-", appName)),
```

Comment:
Not really sure how I feel about this pattern, but it's hard to argue with shortening any of our names

Reply:
i also dont like it, i think we should take the time to make an appname global to all ids so that we dont have to prepend it onto everything. Maybe thats a high value task